spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -235,6 +236,8 @@ def step(  # noqa
         grasp=False,
         place=False,
         max_joint_movement_key="MAX_JOINT_MOVEMENT",
+        max_lin_dist_mobile_gaze=None,
+        max_ang_dist_mobile_gaze=None,
```

Comment:
@jimmytyyang 
Maybe we can directly read this from YAML during initialization, as we dont need to change these variables on every iteration

Reply:
okay, fixed this! thank you!